scripts/cibuild

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env bash
 set -ex
 
-./sbt "project api" assembly
+mkdir -p ${HOME}/.sbt
+
+docker run -v ${HOME}/.ivy2:/root/.ivy2 -v ${HOME}/.sbt:/root/.sbt -v \
```

Comment:
I think we can probably bypass using Docker here and just depend on a Java 8 being present.